pkg/controllers/rollout/controller.go

```diff
@@ -123,6 +123,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req runtime.Request) (runtim
 
 	// fill out all the default values for CRP just in case the mutation webhook is not enabled.
 	fleetv1beta1.SetDefaultsClusterResourcePlacement(&crp)
+	// TODO: revisit to see if we should remove logic for validation
```

Comment:
@ryanzhang-oss i'm thinking of removing this validation. The current validation is very heavy and we should assume validation webhook is enabled.

Reply:
SGTM as the webhook is fail to close